backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/CommitOfManagedBranch.java

```diff
@@ -25,6 +25,12 @@ public String getShortMessage() {
     return coreCommit.getShortMessage();
   }
 
+  @Override
+  @ToString.Include(name = "message")
```

Comment:
Hmmm but `getShortMessage` also is `@Include`d with `name = "message"`... what's gonna be the outcome? ⚔️